pkg/scheduler/framework/runtime/framework.go

```diff
@@ -33,6 +33,7 @@ import (
 	"k8s.io/kube-scheduler/config/v1beta1"
 	"k8s.io/kubernetes/pkg/scheduler/apis/config"
 	"k8s.io/kubernetes/pkg/scheduler/apis/config/scheme"
+	fwk "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
```

Comment:
Rename the variables instead

Reply:
I think it's fine. But it could also be `Runtime`.

Reply:
Lets rename the ```framework``` struct to ```frameworkImpl``` and reserve the name ```framework``` for the interface pkg. This is a local type, the changes should be in this file and the corresponding unit test only.

Reply:
rename the pkg now to ```framework```